modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc

```diff
@@ -0,0 +1,100 @@
+= Using Your Own Prometheus Server to Consume Couchbase Metrics
+:description: With Couchbase 7.x, it's not necessary to write a custom Prometheus Exporter.This page will demonstrate how to enable a Prometheus server to read from Couchbase's Prometheus-ready endpoints.
```

Comment:
Wonder if this is a bit long for the SEO description, maybe shorten this and add more in the abstract?

Reply:
Effectively, only the first couple of dozen words in the description get used by Google.